### PR TITLE
Benchmark data from parsed matches 

### PR DESCRIPTION
**Pull request by Argn0**
https://github.com/odota/web/issues/524

### CODE REVIEW

File: svc/counts.js [hunk 1]
```diff
@@ -225,7 +200,7 @@ function processCounts(match, cb) {
     },
     updateBenchmarks(cb) {
       if (isSignificant(match)) {
-        return updateBenchmarks(match, cb);
+        return benchmarksUtil.updateBenchmarks(match, cb);
```
Comment (howardchung) [inline svc/counts.js]: perhaps we should take this out and just let the separate service do the benchmarking?

File: util/benchmarksUtil.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  hero_healing_per_min(m, p) {
+    return (p.hero_healing / m.duration) * 60;
+  },
+  tower_damage(m, p) {
+    return p.tower_damage;
+  },
+};
+
+const parsedBenchmarks = {
+  stuns(m, p) {
```
Comment (howardchung) [inline util/benchmarksUtil.js]: probably should just combine these and benchmark parsed matches only.  doing all matches for some and parsed matches for others seems too complicated

**Comment by Argn0** [comment]
I made the changes you suggested.

File: config.js [hunk 3]
```diff
@@ -52,6 +52,7 @@ const defaults = {
   SCANNER_PERCENT: 100, // percent of matches to insert from scanner
   PUBLIC_SAMPLE_PERCENT: 10, // percent of public matches to sample in DB
   SCENARIOS_SAMPLE_PERCENT: 50, // percent of parsed matches to sample for scenarios
+  BENCHMARKS_SAMPLE_PERCENT: 100, // percent of parsed matches to sample for benchmarks
```
Comment (howardchung) [inline config.js]: since the number of parsed matches is lower, we should probably increase BENCHMARK_RETENTION_MINUTES to an hour or something

**Comment by Argn0** [comment]
Let me do some tests. I might have broken something.

**Comment by Argn0** [comment]
OK, all is good now.

**Comment by howardchung** [comment]
Could you also add a metric for the queue length to buildStatus (like you did for scenariosQueue?)

**Pull request merged**